larray/inout/xw_reporting.py

```diff
@@ -242,6 +242,18 @@ def add_graph(self, data, title=None, template=None, width=None, height=None):
         height : int, optional
             height of the title item. The current default value is used if None
             (see :py:obj:`~ExcelReport.set_item_default_size`). Defaults to None.
+        **kwargs
+
+          * min_y: int, optional
```

Comment:
I expect users to make other requests in the future and one can easily guess that the number of explicit arguments will increase exponentially. The matplotlib library often uses **kwargs, see for example the [plot](https://matplotlib.org/api/_as_gen/matplotlib.pyplot.plot.html#matplotlib.pyplot.plot) function. 

I find function with lots of explicit arguments ugly (to my personal taste).

Reply:
I can understand if the signature of the function uses **kwargs (I don't think it's a good practice because then PyCharm et al. cannot autocomplete those argument names but that's a trade off I can hear) but what I was talking about is the fact that some arguments are under the `**kwargs` header **in the docstring** which I am pretty sure none of our users understand what it means (and I find ugly)

https://pandas.pydata.org/docs/reference/api/pandas.read_excel.html#pandas.read_excel

Reply:
> I don't think it's a good practice because then PyCharm et al. cannot autocomplete those argument names but that's a trade off I can hear.

After a second thought I give you the point. I usually try to promote the use of auto-complemetation among our users so I have to be consistent with myself. I still fear to end up with two ``add_graph(s)()`` methods with an endless list of explicit arguments but given how we know how users write code, this is maybe (certainly?) the "less worse" solution.

A second point for you: indeed `pandas` have several functions with a long list of explicit arguments.